08-kogito-travel-agency/src/main/java/org/kie/kogito/app/RuleEventListenerConfig.java

```diff
@@ -0,0 +1,15 @@
+package org.kie.kogito.app;
+
+import javax.enterprise.context.ApplicationScoped;
+
+import org.drools.core.config.DefaultRuleEventListenerConfig;
+import org.kie.addons.monitoring.rule.PrometheusMetricsDroolsListener;
```

Comment:
if you change to 0.8.1
```suggestion
import org.kie.kogito.monitoring.rule.PrometheusMetricsDroolsListener;
```